src/cudamatrix/cu-allocator.cc

```diff
@@ -223,6 +223,8 @@ void* CuMemoryAllocator::MallocFromSubregion(SubRegion *subregion,
   block->allocated = true;
   block->t = t_;
   allocated_block_map_[block->begin] = block;
+  allocated_memory_ += (block->end - block->begin);
+  if (allocated_memory_ > max_allocated_memory_) max_allocated_memory_ = allocated_memory_;
```

Comment:
Sorry, a couple small comments.  Would you mind splitting this line?  It looks over the 80-char limit.